release_notes.md

```diff
@@ -20,6 +20,11 @@ This file tracks release notes for the loop client.
   verbose info, giving users a more intuitive view about money paid
   on/off-chain and fees incurred. Use `loop in -v`, `loop out -v`,
   `loop quote in -v` or `loop quote out -v` to view the details.
+* A stripped down version of the Loop server is now provided as a
+  [Docker image](https://hub.docker.com/r/lightninglabs/loopserver). A quick
```

Comment:
We ought to also add a brief overview in Dockerhub or at least point back to the docs.

Reply:
Yes, unfortunately I don't have the permissions to do so. I'm not sure what options we have. I think we can either link a repo and then the main README is displayed (which might not be optimal if we point to the loop repo) or add a static markdown text. Maybe we just link to the correct section in the README and don't add a lot of text otherwise?